### ISSUE TITLE
Problem setting up the material for tree leaf

### ISSUE THREAD

**Issue by Antokolos** [godotengine/godot-blender-exporter#279]
**OS:**

WIN

**Godot version:**

Godot 3.1.1

**Blender version:**

2.79

**Issue description:**
Please take a look at the following Blender model
[bug.zip](https://github.com/godotengine/godot-blender-exporter/files/3595202/bug.zip)
Try to export it to escn and save in the new project
First of all, the branch looks very strange :)
Next, the material for leaves wasn't set up via shader material, and when I'm trying to set it up manually, some weird things happen:
![image](https://user-images.githubusercontent.com/7196697/64607012-6b25f080-d3e9-11e9-9aca-373b33308b56.png)
![image](https://user-images.githubusercontent.com/7196697/64607016-6feaa480-d3e9-11e9-878e-d5d0f2a8ee9d.png)

By the way, I prefer not to use Shader Materials which is created by Godot Blender Exporter and set up blank spatial materials in the Godot instead of it. This is due to the fact that I feel more control over the materials and the exporter also cannot transform complex Blender materials properly anyway. 

To achieve this, I comment out some strings of code from the exporter, as you can see below
![image](https://user-images.githubusercontent.com/7196697/64607284-0ae37e80-d3ea-11e9-8ea8-2112f29a2ff7.png)

It would be super nice if Godot Blender Exporter had and additional checkbox for people like me as well :)

**Minimal reproduction project:**

[leaf_bug.zip](https://github.com/godotengine/godot-blender-exporter/files/3595207/leaf_bug.zip)

**Comment by Antokolos** [comment]
OK, the problem with the leaves was in the incorrect UV unwrap, so we fixed the main problem for our use case.
But maybe this model can help you fixing some other things.

**Comment by Jason0214** [comment]
>  I prefer not to use Shader Materials which is created by Godot Blender Exporter and set up blank spatial materials in the Godot instead of it.

Good point, master branch already default to export SpatialMaterial, I may need to cherry pick [it](https://github.com/godotengine/godot-blender-exporter/commit/a55346d1376237ece258a7220897e81b1e445e2d) to 2.79 branch.
